src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/jpa/entity/SampleData.kt

```diff
@@ -94,6 +94,37 @@ class SampleData {
       )
     }
 
+    fun sampleReferralSummary(
```

Comment:
sorry - this isn't clear in the code, but this class is deprecated in favour of the newer factory classes. we shouldn't be adding new code to it.